scripts/predict.py

```diff
@@ -10,36 +11,48 @@
 
 @click.command()
 @click.option('--mask_path', required=True,
-              help='Adress of json file of the mask')
+              help='Path of json file of the mask')
 @click.option('--picture_path', required=True,
-              help='Adress of picture to be predicted')
+              help='Path of picture to be predicted')
 @click.option('--model_path', required=True,
-              help='Path where pretrained model')
+              help='Path where pretrained model is located')
 @click.option('--output', default=None,
-              help='Name of output file for predicted output')
-def load_and_predict(mask_path, picture_path, model_path, output):
+              help='Name of file for predicted output')
+@click.option('--img_suffix', default='.png',
+              help='Suffix of predicted image')
+def load_and_predict(mask_path, picture_path, model_path, output, img_suffix):
     for file in (mask_path, picture_path, model_path):
```

Comment:
@Adman You can pass `type=click.Path(exists=True)` parameter to click arguments and click will handle existence of files so you do not have to check it by yourself. http://click.pocoo.org/5/parameters/#parameter-types and http://click.pocoo.org/5/arguments/#file-path-arguments